--- cpu-g-0.16.2/bin/cpu-g.orig	2020-04-11 12:46:16.000000000 +0000
+++ cpu-g-0.16.2/bin/cpu-g	2020-04-11 16:46:47.675678903 +0000
@@ -32,9 +32,9 @@ except Exception as e:
 from gi.repository import Gtk
 
 if __name__ == '__main__':
-    if __file__.startswith('/opt/extras.ubuntu.com/cpu-g') or\
-            os.getcwd().startswith('/opt/extras.ubuntu.com/cpu-g'):
-        sys.path.insert(1, '/opt/extras.ubuntu.com/cpu-g/share/cpu-g')
+    if __file__.startswith('/usr/share/cpu-g') or\
+            os.getcwd().startswith('/usr/share/cpu-g'):
+        sys.path.insert(1, '/usr/share/share/cpu-g')
     else:
         sys.path.insert(1,
                         os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../src')))
--- cpu-g-0.16.2/src/comun.py.orig	2020-04-11 12:46:16.000000000 +0000
+++ cpu-g-0.16.2/src/comun.py	2020-04-11 16:49:31.890104148 +0000
@@ -38,11 +38,11 @@ APPNAME = 'CPU-G'
 
 # check if running from source
 if is_package():
-    ROOTDIR = '/opt/extras.ubuntu.com/cpu-g/share'
-    LANGDIR = os.path.join(ROOTDIR, 'locale-langpack')
+    ROOTDIR = '/usr/share'
+    LANGDIR = os.path.join(ROOTDIR, 'locale')
     APPDIR = os.path.join(ROOTDIR, APP)
     CHANGELOG = os.path.join(APPDIR, 'changelog')
-    ICONDIR = os.path.join(ROOTDIR, 'icons')
+    ICONDIR = os.path.join(ROOTDIR, APP)
     ICON = os.path.join(ICONDIR, 'cpu-g.png')
     LOGOSDIR = os.path.join(APPDIR, 'logos')
     DISTROSDIR = os.path.join(APPDIR, 'distros')
