--- cpu-g	2009-10-15 23:17:40.000000000 +0200
+++ cpu-g	2011-07-04 23:15:32.557052185 +0200
@@ -52,7 +52,7 @@
         model = self.cpuinfo("model", core)
         vendor = self.cpuinfo("vendor", core)
         
-        path = 'data/logos/'
+        path = '/usr/share/cpu-g/data/logos/'
         
         if vendor == 'AMD':
             label = 'amd.png'
@@ -228,7 +228,7 @@
     """Description"""
     
     def __init__(self):
-        self.gladefile = "cpu-g.glade"
+        self.gladefile = "/usr/share/cpu-g/cpu-g.glade"
         self.wTree = gtk.Builder()
         self.wTree.add_from_file(self.gladefile)
         
