--- cpu-g-0.16.2/src/test.py.orig	2020-04-11 12:46:16.000000000 +0000
+++ cpu-g-0.16.2/src/test.py	2020-04-11 16:11:24.444712429 +0000
@@ -24,8 +24,7 @@
 from gi.repository import Gtk
 
 from matplotlib.figure import Figure
-from matplotlib.backends.backend_gtk3agg import FigureCanvasGTK3Agg as
-FigureCanvas
+from matplotlib.backends.backend_gtk3agg import FigureCanvasGTK3Agg as FigureCanvas
 
 win = Gtk.Window()
 win.connect("delete-event", Gtk.main_quit)
